CWE/inventions/culture_inventions.txt

```diff
@@ -282,29 +298,31 @@ socialist_contemporary_science = {
 }
 # pstn_network
 satellite = {
-        limit = { pstn_network = 1 }
-        chance = {
-                base = 2
-                modifier = {
-                    factor = 12
-					OR = { 
-						tag = USA 
-						tag = RUS 
-					} 
-                }
-                modifier = {
-                        factor = 2
-                        is_greater_power = yes
-                }
-        }
-        effect = {
-            permanent_prestige = 2
-			education_efficiency = 0.1
-        }
+	limit = { pstn_network = 1 }
+	news = no
+	chance = {
+		base = 2
+		modifier = {
+			factor = 12
+			OR = { 
+				tag = USA 
+				tag = RUS 
+			} 
```

Comment:
Should not have boosts based on tags as that would be railroading. Better to check for the space agency modifier instead.

Reply:
Alright.